### PR TITLE
DOC-2223: TinyMCE 6.8.1 community release.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2223

Changes:
* TinyMCE 6.8.1 community release.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `.api-version` bump

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.8.1 - 2023-11-29
+
+=== Improved
+* Colorpicker now includes the Brightness/Saturation selector and hue slider in the keyboard navigable items.
+
+=== Fixed
+* The functions `schema.isWrapper` and `schema.isInline` didn't exclude element names that started with `#` those should not be considered elements.
```
Comment (spocke) [inline modules/ROOT/pages/changelog.adoc:13]: There where some alterations to the master branch on this one: https://github.com/tinymce/tinymce/blob/master/modules/tinymce/CHANGELOG.md

I could merge master into develop but we will do that tomorrow anyway when we do the release.

**Review by spocke — APPROVED** [review]

File: changelog.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 
 ### Unreleased
 
+### 2023-11-29 (unreleased)
```
Comment (kemister85) [inline changelog.md:9]:
```suggestion
### 2023-11-29
```

File: modules/ROOT/pages/changelog.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:13]:
```suggestion
* The functions `schema.isWrapper` and `schema.isInline` did not exclude node names that started with `#` which should not be considered as elements. 
```

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**